include/openenclave/internal/sgxcreate.h

```diff
@@ -65,6 +65,11 @@ struct _oe_sgx_load_context
     /* EEID data needed during enclave creation */
     oe_eeid_t* eeid;
 #endif
+
+    uint8_t  enclave_config_id[64];
```

Comment:
Consider creating a struct that matches the settings struct, and have a `bool` to check is the `config_data` is set.
```
    struct
    {
        uint8_t config_id[32];
        uint16_t config_svn;
        bool ignore_if_unsupported;
    } config_data;
    bool is_config_data_set;
```

Or even reuse the setting struct
```
typedef oe_enclave_setting_config_data oe_config_data_t;
...
struct _oe_sgx_load_context
{
   ...
   oe_config_data_t config_data;
   bool is_config_data_set;
}
```